checks.d/teamcity.py

```diff
@@ -21,7 +21,7 @@ def __init__(self, name, init_config, agentConfig, instances=None):
         # Keep track of last build IDs per instance
         self.last_build_ids = {}
 
-    def _initialize_if_required(self, instance_name, server, build_conf):
+    def _initialize_if_required(self, instance_name, server, build_conf, ssl_validation):
```

Comment:
@irabinovitch sorry, missed the original notification. I added it in. Do you want me to rebase the PR against current master?